awspytools/DynamoDBDataStore.py

```diff
@@ -37,7 +37,12 @@ class DynamoDBDataStore(object):
     serializer = TypeSerializer()
     deserializer = TypeDeserializer()
 
-    def __init__(self, table_name, hash_key='PK', sort_key='SK', use_default_index_keys=True, endpoint_url=None):
+    def __init__(self, table_name,
+                 hash_key='PK',
+                 sort_key='SK',
+                 use_default_index_keys=True,
+                 client=None,
+                 endpoint_url=None):
```

Comment:
I think we should move client to the last argument in the off chance that a project uses positional arguments only.